packages/terra-date-picker/tests/nightwatch/components/DatePickerOnChange.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import DatePicker from '../../../lib/DatePicker';
+
+const handleOnChange = (selectedDate) => {
+  window.console.log('**handleOnChange** The selected date is: ', selectedDate);
+};
+
+const DatePickerOnChange = () => (
+  <div>
```

Comment:
Nope, I think it was copied/pasted from another component. Removed: https://github.com/cerner/terra-core/commit/6f069c2d15d4fef6e983c0e62b91a8b053041e48